onboard/merge_iterator_test.go

```diff
@@ -23,6 +23,8 @@ func TestMergeIterator(t *testing.T) {
 			InvIt: testutils.NewFakeEntryIterator([]*rocks.EntryRecord{
 				{Path: "a/b/c"},
 				{Path: "a/d/c"},
```

Comment:
```suggestion
				{Path: "a/c/d"},
```
(I want a dupe in something included by the prefix!)

Reply:
You got it.